### PR TITLE
Fix wrong twist in odometry messages

### PR DESCRIPTION
**Pull request by mateusz-lichota**
As discussed in #283

### CODE REVIEW

**Comment by mateusz-lichota** [comment]
@wouter-heerwegh care to take a look at the changes?

**Review by wouter-heerwegh — CHANGES_REQUESTED** [review]
Sorry, I didn't know you couldn't see my review.

File: ros2/src/fsds_ros2_bridge/src/airsim_ros_wrapper.cpp [hunk 1]
```diff
@@ -319,11 +322,11 @@ nav_msgs::msg::Odometry AirsimROSWrapper::get_odom_msg_from_airsim_state(const m
...
-    odom_enu_msg.twist.twist.linear.x = car_state.kinematics_estimated.twist.linear.x();
-    odom_enu_msg.twist.twist.linear.y = car_state.kinematics_estimated.twist.linear.y();
-    odom_enu_msg.twist.twist.linear.z = car_state.kinematics_estimated.twist.linear.z();
-    odom_enu_msg.twist.twist.angular.x = car_state.kinematics_estimated.twist.angular.x();
-    odom_enu_msg.twist.twist.angular.y = car_state.kinematics_estimated.twist.angular.y();
+    odom_enu_msg.twist.twist.linear.x = car_state.speed;
+    odom_enu_msg.twist.twist.linear.y = 0;
+    odom_enu_msg.twist.twist.linear.z = 0;
+    odom_enu_msg.twist.twist.angular.x = 0;
+    odom_enu_msg.twist.twist.angular.y = 0;
```
[the remark this answers is not in the document]
Comment (mateusz-lichota) [reply ros2/src/fsds_ros2_bridge/src/airsim_ros_wrapper.cpp:329]: linear.z should definitely be 0, as the car won't move int the up/down direction. Same for angular.x and angular.y: for those to be meaningfully nonzero the car would have to roll over or jump of a ramp, both of which are not scenarios we expect, support or care about here. I agree with you that linear.y shouldn't always be set to zero, and I'll look into how we can achieve that.

[the remark this answers is not in the document]
Comment (wouter-heerwegh) [reply ros2/src/fsds_ros2_bridge/src/airsim_ros_wrapper.cpp:329]: The car will roll and pitch quite a lot, a lot more than our actual car. I agree that the measurement will most likely not be very useful, but I think it is still better to add these for completeness.

**Comment by mateusz-lichota** [comment]
Hey @wouter-heerwegh, I've implemented the computation for sideways velocity using the yaw angle to compute it from map frame velocities. However, transforming the angular velocities is quite difficult, and I think is not necessary, because when the car rolls and pitches what matters to people is the actual roll and pitch, and very rarely the roll rate or pitch rate. I honestly can't think of any practical use for those values, but if any is found someone more experienced than me can write the appropriate transformation code.

**Review by wouter-heerwegh — APPROVED** [review]
LGTM, thank you very much!

**Pull request merged**